apps/web/src/components/ExperienceFormFields/AwardFields.tsx

```diff
@@ -34,7 +34,15 @@ const AwardFields = ({ labels }: SubExperienceFormProps) => {
             label={labels.awardTitle}
             name="awardTitle"
             type="text"
-            rules={{ required: intl.formatMessage(errorMessages.required) }}
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+              maxLength: {
+                message: intl.formatMessage(errorMessages.overCharacterLimit, {
+                  value: 256,
+                }),
+                value: 255,
+              },
+            }}
```

Comment:
Could we add this directly into the `<Input />` component so we can prevent this error across the app?

Reply:
Something like this:
```tsx
// packages/forms/src/components/Input.tsx line 77
 {...register(name, {
          maxLength: {
            message: intl.formatMessage(errorMessages.overCharacterLimit, {
              value: 256,
            }),
            value: 255,
          },
          ...rules,
          onBlur: normalizeInput,
        })}
 ...
```

Reply:
Decision from standup: we like this idea, will do it in a separate issue